src/App.vue

```diff
@@ -8,6 +8,7 @@
       class="x-modal"
       modalId="x-components-app"
       :animation="collapseFromTop"
+      dir="ltr"
```

Comment:
I thought you were going to use the variable coming from the `snippetConfig`, even if I still think that thid configuration should be done at the `html` or `body` level

Reply:
Done.

Agree. This configuration should be done in the html / body but we can't control what the user of the archetype does. I've upload a new approach. It gives priority to what is set in the html, then the body, then the snippetConfig and 'ltr' as fallback.

But as we've talked I'd not waste more time if we're gonna focus on the RTL in the future.